client/src/components/ExpenseAdder/ExpenseAdder.styles.ts

```diff
@@ -0,0 +1,81 @@
+import styled from "styled-components";
+import { motion } from "framer-motion";
+
+interface Props {
```

Comment:
Can you change this to be more specific? Props is just kind of vague.  Maybe ExpenseProps? Or something.